scripts/upgrade

```diff
@@ -38,6 +38,12 @@ fi
 chmod -R o-rwx "$install_dir"
 chown -R "$app:www-data" "$install_dir"
 
+# clear the smarty cache
```

Comment:
Did some testing, this part (41-45) probably needs to be above the chmod and chown steps in lines 38 and 39, otherwise the compiled folder gets owned by root and Friendica nor www-data can't write to that directory.